code-challenges.js

```diff
@@ -22,11 +57,37 @@ var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
 
+const rem = (arr) => {
+  
+  let newArray = []
+
+  for (let i = 0; i < arr.length; i++) {
+    let remainder = 0
+    if(typeof arr[i] === 'number'){
+    remainder = arr[i] % 3
+    // console.log(remainder)
+    newArray.push(remainder)
+    // return arr[i]
+    }
+  }
+  return newArray
```

Comment:
This looks awesome too! :) Again I think it would be nice to see some pseudo code here. Also, it would be interesting to see how we could use .filter and .map to solve this problem in order to make it a bit more concise :)